client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1449,6 +1454,10 @@
   "messages.confirm-delete-assets_one": "This will permanently remove this asset.",
   "messages.confirm-delete-assets_other": "This will permanently remove the {{count}} selected assets.",
   "messages.confirm-delete-bundled-item": "This will remove this item from the bundle",
+  "messages.confirm-delete-campaigns_few": "This will permanently remove {{count}} campaign",
+  "messages.confirm-delete-campaigns_many": "This will permanently remove {{count}} campaigns",
+  "messages.confirm-delete-campaigns_one": "This will permanently remove 1 campaign",
+  "messages.confirm-delete-campaigns_other": "This will permanently remove {{count}} campaigns",
```

Comment:
I have a tangent thought that it would be awesome to have a second dynamic value so we don't need so many similar delete confirmation messages

ie:
`This will permanently remove {{count}} {{name}}`

Reply:
That's a cool idea, it seemed pretty wasteful to me too, but probably beyond the scope of this issue as I'd need to change it in all the places that use it. Do you want to make a re-factor issue to address this more widely?

Reply:
No worries I can make the issue and see if it gains traction.

Did not mean to imply should be added in this issue - was just jotting down my thoughts :)